apps/antalmanac/src/components/dialogs/AddSchedule.tsx

```diff
@@ -46,7 +53,14 @@ function AddScheduleDialog(props: ScheduleNameDialogProps) {
 
     const submitName = useCallback(() => {
         addSchedule(name);
-        setName(`Schedule ${AppStore.getScheduleNames().length + 1}`);
+        setName(
+            `${
+                termName +
+                (AppStore.getScheduleNames().filter((name) => name.includes(termName)).length == 0
+                    ? ''
+                    : '(' + AppStore.getScheduleNames().filter((name) => name.includes(termName)).length + ')')
+            }`
```

Comment:
Please refactor this into a dedicated function. The implementation would be something like this:

```ts
function getDefaultScheduleName() {
  const termName = termData[0].shortName.replaceAll(' ', '-');
  const countSameScheduleNames = scheduleNames.filter((name) => name.includes(termName)).length;
  return `${termName + (countSameScheduleNames == 0 ? '' : '(' + countSameScheduleNames + ')')}`
}
```

You can put it in the `AppStore` so it has access to the current state of the schedule. Inlining the code pollutes the function and can lead to inconsistencies between name calculations